### PR TITLE
Improve logging and update README

### PR DESCRIPTION
**Pull request by cbartlett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,100 @@
...
 ```
 
-## Building for all platforms
+### Linux
+
+You will need to make the binary executable before running it:
 
 ```
-make
+$ chmod +x trackssl-agent-linux
```
Comment (alibby) [inline README.md:33]: we should do this during the build process.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,100 @@
...
 
-## Windows Development
+### Mac
 
-Building:
+You will need to make the binary executable before running it:
 
 ```
-go build -o agent.exe
+$ chmod +x trackssl-agent-mac
```
Comment (alibby) [inline README.md:47]: All the binaries will build by simply running `make`

**Review by alibby — CHANGES_REQUESTED** [review]
Just a few thoughs included.

File: README.md (hunk 1 above)
Comment (cbartlett) [reply README.md:33]: Agreed, but will they retain that attribute when uploaded to releases in GitHub? Cuz people generally are not building it themselves, they are downloading the binary.

File: README.md (hunk 2 above)
Comment (cbartlett) [reply README.md:47]: Not sure what you mean, I do have that at the bottom of the README.

File: README.md (hunk 1 above)
Comment (alibby) [reply README.md:33]: > Agreed, but will they retain that attribute when uploaded to releases in GitHub? Cuz people generally are not building it themselves, they are downloading the binary.

Yeah, that just occurred to me independently.

**Review by alibby — APPROVED** [review]

**Pull request merged**